### PR TITLE
Fix for data is dropped if message is not present in notification

### PR DESCRIPTION
**Pull request by manasdk**

### CODE REVIEW

**Comment by lakshmi-kannan** [comment]
As kami would say, no tests!!!

**Comment by manasdk** [comment]
> As kami would say, no tests!!!

Fixed b3965bc5996505481e5c1545e4e5055b5e6d3223

File: st2common/st2common/models/api/notification.py [hunk 1]
```diff
@@ -94,7 +94,7 @@ def _from_model_sub_schema(notify_sub_schema_model):
 
         if getattr(notify_sub_schema_model, 'message', None):
             notify_sub_schema['message'] = notify_sub_schema_model.message
-        if getattr(notify_sub_schema_model, 'message', None):
+        if getattr(notify_sub_schema_model, 'data', None):
```
Comment (Kami) [inline st2common/st2common/models/api/notification.py]: Good catch.

**Comment by Kami** [comment]
Good catch, LGTM, +1.

(I believe a user on Slack had a similar issue a couple of days ago)

**Pull request merged**